test/concerns/hideable_test.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+class HideableTest < ActiveSupport::TestCase
+  test "scope hidden, not_hidden" do
+    hidden_book = Book.create(title: "The Art of War", hidden: true)
+    not_hidden_book = Book.create(title: "Alphabetic writing", hidden: false)
+
+    assert_equal 1, Book.hidden.count
+    assert_equal 1, Book.not_hidden.count
+    assert_equal true, hidden_book.hidden?, "hidden_book is not hidden"
+    assert_equal false, not_hidden_book.hidden?, "not_hidden_book is hidden"
+  end
+
+  test "hide!" do
```

Comment:
@nkumeiko 
We use common ruby agreement `#featured?` for instance methods and `.featured?` for class methods, to name tests. See http://betterspecs.org/ru/#describe, and refactor for all tests.